tests/behat/features/bootstrap/FeatureContext.php

```diff
@@ -17,15 +17,17 @@ class FeatureContext implements Context
      */
     public static function setupDatabase($scope)
     {
-        // Copy Fixture to guest
-        shell_exec("cat ./fixtures/events.sql | vagrant ssh -c 'cat -> /tmp/events.sql'");
+        // Generate test databases
```

Comment:
I'd like some feedback that the fixtures are about to be loaded. Maybe echo a status message? Something like:

`Running before suite hook\n\nSetting up test databases`

Reply:
And some tests seem to have stopped working. Before your changes all tests except for the sso tests where green. Please investigate what might be the problem here.

Reply:
Added the extra feedback and fixed the tests for the institution configuration